cypress/e2e/DoenetML/tagSpecific/mathinput.cy.js

```diff
@@ -7602,4 +7602,31 @@ describe('MathInput Tag Tests', function () {
 
   });
 
+  it('mathinput updates not messed up with invalid child logic containing a composite', () => {
+    cy.window().then(async (win) => {
+      win.postMessage({
+        doenetML: `
+      <ol>
+        <math name="m">x</math> $m
```

Comment:
Opened an issue about how we want to surface errors/warnings to doenetML authors for these types of problems. https://github.com/Doenet/DoenetTools/issues/1956